database/seeders/SemesterSeeder.php

```diff
@@ -15,17 +15,24 @@ class SemesterSeeder extends Seeder
      */
     public function run()
     {
-        // TODO: there could be more semesters
-        $semester = Semester::create([
-            'year' => 2019,
-            'part' => 2,
+        $semester = Semester::updateOrCreate([
+            'year' => 2015,
+            'part' => 1,
         ]);
 
+        while ($semester != Semester::next()) {
+            // This creates the semester if it does not exists
+            $semester = $semester->succ();
+        }
+
         $users = User::all();
 
-        //TODO: this could be more random
-        foreach ($users as $key => $user) {
-            $user->allSemesters()->attach($semester, ['status' => Semester::ACTIVE]);
+        $semesters = Semester::all();
+        foreach ($semesters as $semester) {
+            foreach ($users as $user) {
+                $status = array_rand(Semester::STATUSES);
+                $user->allSemesters()->attach($semester, ['status' => Semester::STATUSES[$status]]);
+            }
```

Comment:
We need this only for collegists, not all users.

Reply:
Right, thanks!